sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -106,16 +107,20 @@ def __init__(self, key, credential, **kwargs):
             self._key = None
             self._key_id = parse_key_vault_id(key)
             self._keys_get_forbidden = None  # type: Optional[bool]
+        elif self._jwk:
+            self._key_id = key.get("kid", "")
```

Comment:
By defaulting to an empty string, we can leave the `key_id` property return type as a string. This feels a little questionable though -- what do you think?

Reply:
My gut says let it be None, and document that it may be None when the client was constructed by `from_jwk`.